command/report/report.go

```diff
@@ -25,9 +27,22 @@ type ReportOptions struct {
 func NewCmdReport() *cobra.Command {
 	opts := ReportOptions{}
 
+	c := color.New(color.FgCyan, color.Bold)
+	y := color.New(color.FgYellow, color.Bold)
+	doc := heredoc.Docf(`
+		Report artifacts to DeepSource.
+
+		Use %[1]s to specify the analyzer, for example:
+		%[2]s
+
+		Use %[3]s to specify the value of the artifact:
+		%[4]s
+		`, y.Sprintf("--analyzer"), c.Sprintf("deepsource report --analyzer python"), y.Sprintf("--value"), c.Sprintf("deepsource report --key value"))
```

Comment:
Need more examples for this. Any suggestions?

Reply:
`deepsource report --analyzer go --value-file coverage.out`

`deepsource report --analyzer javascript --value-file coverage/cobertura-coverage.xml`